gensim/models/hdpmodel.py

```diff
@@ -56,6 +56,21 @@ def dirichlet_expectation(alpha):
     return(sp.psi(alpha) - sp.psi(np.sum(alpha, 1))[:, np.newaxis])
 
 
+def get_random_state(seed):
```

Comment:
I'll move the common `ldamodel` and `hdpmodel` methods to the respective `utils` and `matutils` files after this PR is merged.